backend/kesaseteli/applications/tests/test_excel_export.py

```diff
@@ -352,6 +355,48 @@ def employer_summer_voucher_sorting_key(voucher: EmployerSummerVoucher):
                 ), excel_field.title
 
 
+@pytest.mark.django_db
+@override_settings(NEXT_PUBLIC_MOCK_FLAG=False)
+@pytest.mark.parametrize(
+    "employer_summer_voucher_creation_date,sum_field_value",
+    [
+        (date(2021, 6, 1), "325"),
```

Comment:
Trivial, but the number doesn't feel like a price when it's a string literal, because there is no euromark.

Reply:
I'll add something in another PR to make this a bit clearer.

Reply:
→ https://github.com/City-of-Helsinki/yjdh/pull/2754